README.md

```diff
@@ -7,8 +7,181 @@ A collection of named Excel/Google Sheets formulas using LET and LAMBDA function
 <!-- AUTO-GENERATED CONTENT START -->
 <!-- This section is automatically generated by generate_readme.py -->
 
+### Quick Reference
+
 - **[UNPIVOT](unpivot.yaml)** - Transforms wide-format data into long-format (tidy data) by unpivoting specified columns into attribute-value pairs.
 
+### Detailed Formulas
+
+<details>
+<summary><strong>UNPIVOT</strong></summary>
+
+**Version**: `1.0.0`
+
+**Description**
+
+```
+Transforms wide-format data into long-format (tidy data) by unpivoting specified columns into attribute-value pairs.
+```
+
+**Parameters**
+
+- `data`
+
+```
+Input range including headers (first row must contain column names)
+```
```

Comment:
Now that the code fences are left-aligned, that fact that the parameters name, `data` in this case, is not left-aligned because it is a bullet list item, makes it a bit confusing. If the code fence were indented once step to the right it would make sense, or alternatively, we just have to make the parameter name also non-indentend.